main.go

```diff
@@ -302,13 +200,44 @@ func ensureFastlaneVersionAndCreateCmdSlice(forceVersion, gemfilePth string) ([]
 }
 
 func main() {
-	configs := createConfigsModelFromEnvs()
+	var configs Configs
+	if err := stepconf.Parse(&configs); err != nil {
+		fail("Issue with input: %s", err)
+	}
 
+	stepconf.Print(configs)
+
+	//
+	// Validate inputs
+	if configs.IpaPath == "" && configs.PkgPath == "" {
+		fail("no IpaPath nor PkgPath parameter specified")
+	}
+
+	if configs.AppID == "" && configs.BundleID == "" {
+		fail("no AppID or BundleID parameter specified")
```

Comment:
same as above

Reply:
Fixed in:
https://github.com/bitrise-io/steps-deploy-to-itunesconnect-deliver/pull/61/commits/e8197e70de7fab16f2e93b03a86a88197ba89ef0